docs/_docs/0.8.0/0_3_migration_guide.md

```diff
@@ -0,0 +1,72 @@
+---
+version: 0.8.0
+title: Migration Guide
+keywords: hudi, migration, use case
+permalink: /docs/migration_guide.html
+summary: In this page, we will discuss some available tools for migrating your existing table into a Hudi table
+last_modified_at: 2019-12-30T15:59:57-04:00
+---
+
+Hudi maintains metadata such as commit timeline and indexes to manage a table. The commit timelines helps to understand the actions happening on a table as well as the current state of a table. Indexes are used by Hudi to maintain a record key to file id mapping to efficiently locate a record. At the moment, Hudi supports writing only parquet columnar formats.
```

Comment:
is this something new you add w/ 0.8.0 or is this a copy of already existing content? this is definitely useful. just wanted to see if I have to do line by line reviews ?

Reply:
May I know from where this page is linked? https://hudi.apache.org/docs/migration_guide.html
in other words, how to navigate to this page from our home page.